tests/Integrations/WordPress/V4_8/CommonScenariosTest.php

```diff
@@ -59,28 +59,43 @@ public function provideSpecs()
                         'http.url' => 'http://localhost:9999/simple',
                         'http.status_code' => '200',
                         'integration.name' => 'wordpress',
+                    ])->withChildren([
```

Comment:
I had to migrate WP to flame graph testing as sandboxing mysqli all the orders where messed up